notebooks/206-vision-paddlegan-anime/206-vision-paddlegan-anime.ipynb

```diff
@@ -0,0 +1,614 @@
+   "execution_count": null,
+   "id": "adc56b11",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Step 1. Load the image and convert to RGB\n",
+    "image = cv2.cvtColor(cv2.imread(str(image_path), flags=cv2.IMREAD_COLOR), cv2.COLOR_BGR2RGB)\n",
+    "\n",
+    "# Step 2. Transform the image (only resize and transpose are still required)\n",
+    "resized_image = cv2.resize(image, (target_width, target_height))\n",
+    "input_image = resized_image.transpose(2, 0, 1)[None, :, :, :]\n",
+    "\n",
+    "# Step 3. Do inference.\n",
+    "result_ir = exec_net.infer({input_key: input_image})\n",
+    "\n",
+    "# Step 4. Convert the inference result to an image following the same steps as \n",
+    "# PaddleGAN's predictor.run() function \n",
+    "result_image_ir = (result_ir[output_key] * 0.5 + 0.5)[0] * 255\n",
+    "result_image_ir = result_image_ir.transpose((1, 2, 0))\n",
+    "\n",
+    "# Step 5. Resize the result image\n",
+    "result_image_ir = cv2.resize(result_image_ir, image.shape[:2][::-1])\n",
+    "\n",
+    "# Step 6. Adjust the brightness\n",
+    "result_image_ir = adjust_brightness(result_image_ir, image)\n",
+    "\n",
+    "# Step 7. Save the result image\n",
+    "anime_fn_ir = image_path.with_name(image_path.stem + \"_anime_ir\").with_suffix(\".jpg\")\n",
+    "if cv2.imwrite(str(anime_fn_ir), result_image_ir[:, :, (2, 1, 0)]):\n",
+    "    print(f\"The anime image was saved to {anime_fn_ir}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3d62495d-cb89-44f1-9e43-d946fb1a27d3",
+   "metadata": {},
+   "source": [
+    "**Show Inference Results**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5cc44b64",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots(1, 3, figsize=(25, 15))\n",
+    "ax[0].imshow(image)\n",
+    "ax[1].imshow(result_image_pg)\n",
+    "ax[2].imshow(result_image_ir)\n",
+    "ax[0].set_title(\"Image\")\n",
+    "ax[1].set_title(\"PaddleGAN result\")\n",
+    "ax[2].set_title(\"OpenVINO IR result\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "063ce658-b757-45da-9c7d-929b4be89e43",
+   "metadata": {},
+   "source": [
+    "## Performance Comparison\n",
+    "\n",
+    "Measure the time it takes to do inference on an image. This gives an indication of performance. For more accurate benchmarking, use the [OpenVINO benchmark tool](https://github.com/openvinotoolkit/openvino/tree/master/inference-engine/tools/benchmark_tool)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f5172a4a-f6ef-4142-8917-6c877706b8d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start = time.perf_counter()\n",
+    "exec_net.infer(inputs={input_key: input_image})\n",
+    "end = time.perf_counter()\n",
+    "time_ir = end - start\n",
+    "print(\n",
+    "    f\"IR model in Inference Engine/CPU: {time_ir:.3f} \" f\"seconds per image, FPS: {1/time_ir:.2f}\"\n",
```

Comment:
Can we have a GPU performance check too? Maybe commented out like the PyTorch fastseg. I switched to using GPU device and it works. This is the performance (but still says "CPU" in the printed text)
![image](https://user-images.githubusercontent.com/15709723/123476411-87d8eb80-d5b1-11eb-8247-a6b43a6c3346.png)